kotlin/workflow-runtime/src/main/java/com/squareup/workflow/internal/LifetimeTracker.kt

```diff
@@ -43,11 +43,11 @@ internal class LifetimeTracker<FactoryT, KeyT : Any, DisposableT>(
   private val factoriesByKey = mutableMapOf<KeyT, FactoryT>()
   private val disposablesByKey = mutableMapOf<KeyT, DisposableT>()
 
-  val lifetimes: Map<FactoryT, DisposableT>
+  val lifetimes: List<Pair<FactoryT, DisposableT>>
```

Comment:
Doesn't make sense to think of `FactoryT` as a key, and this isn't actually used as a map anywhere other than unit tests, it's intended to be, and used as, a list of pairs, so let's call a spade a spade.